docs/source/Support/bskReleaseNotes.rst

```diff
@@ -52,6 +52,7 @@ Version |release|
   of the Basilisk repo due to some large files being stored in the GitHub large file storage
   system.
 - Added a new helper function to convert C arrays to ``Eigen::MRPd`` and vice-versa inside ``avsEigenSupport``.
+- Added the ability to set an inertial heading in the :ref:`boreAngCalc` module, along with some quality-of-life improvements.
```

Comment:
unclear what "quality of life" means, be more specific what behavior changes were done?

Reply:
Done